content/docs/command-reference/version.md

```diff
@@ -12,20 +12,18 @@ usage: dvc version [-h] [-q | -v]
-| `Python version`                            | Version of Python used in the environment where DVC is initialized                                                                                                    |
-| `Platform`                                  | Information about the operating system of the machine                                                                                                                 |
-| [`Binary`](#what-we-mean-by-binary)         | Shows whether DVC was installed from a package or from a binary release                                                                                               |
-| `Package`                                   | Name of the package manager used to install DVC if any (`pip`, `conda`, etc)                                                                                          |
-| `Supported remotes`                         | Remote types that have all needed dependencies installed for                                                                                                          |
-| `Cache`                                     | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported between the <abbr>workspace</abbr> and the <abbr>cache</abbr> |
-| `Filesystem type`                           | Shows the filesystem type (eg. ext4, FAT, etc.) and mount point of the cache and <abbr>workspace</abbr> directories                                                   |
-| `Repo`                                      | Shows whether we are in a DVC repo and/or Git repo                                                                                                                    |
-
-> If `dvc version` is executed outside a DVC project, no `Cache` info is
-> printed, and the `Filesystem type` output is of the current working directory.
+| Line                                        | Detail                                                                                                                                                               |
+| ------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`DVC version`](#components-of-dvc-version) | Version of DVC, and name of the binary or package manager (`pip`, `conda`, etc.) used to install DVC (along with a Git commit hash in case of a development version) |
+| `Platform`                                  | Version of Python in the environment where DVC is initialized, and information about the operating system                                                            |
+| `Supports`                                  | Types of [remote storages](/doc/command-reference/remote/add#supported-storage-types) supported by current DVC (their required dependencies are installed)           |
```

Comment:
> supported by current DVC

Bad grammar.

Reply:
I see this is changed but I can't detect the changes in https://github.com/iterative/dvc.org/pull/1814/files#diff-b78fd7ff9804c0395699483d2b3eaba7 ... Will resolve and reopen I guess.